### ISSUE TITLE
sd throw errors when run on empty files

### ISSUE THREAD

**Issue by Porkepix** [chmln/sd#74]
Sometimes, user need to run `sd` on a large collection of file, eg. using `fd`or `find`.
For evey file in this collection that's empty, `sd`will throw the following error:
```
Error processing foobar: memory map must have a non-zero length
```

**Comment by chmln** [comment]
@Porkepix does this happen with the latest release?

**Comment by Porkepix** [comment]
Yes, version `0.7.2`.

**Comment by chmln** [comment]
Thanks @Porkepix, a fix is out in v0.7.4

**Comment by Porkepix** [comment]
Thanks!
